src/utils/constants.ts

```diff
@@ -71,4 +71,8 @@ export const supportedWallets = [
     name: "Metamask",
     url: "https://metamask.io/",
   },
+  {
+    name: "Crypto.com Wallet Extension",
```

Comment:
Changed name field to better fit available width

```suggestion
    name: "Crypto.com Wallet",
```